### DISCUSSION TITLE
Add a way for the popup to open on the screen the icon was clicked on.

### DISCUSSION THREAD

**Discussion by ProbablyIdiot** [File-New-Project/EarTrumpet#1602] [Ideas]
Just realised, on my dual-monitor setup, if i click the icon on my second monitor, it still opens the popup on my primary display. I feel like atleast being able to toggle this behaviour would be very useful. Sorry if this is a duplicate, I had a quick look, but couldn't see anything about this already.

**Comment by riverar** [comment]
@ProbablyIdiot Click which icon? The system tray icon? The system tray doesn't appear on multiple displays so am a bit confused. Are you using third party software that provides this capability?

**Comment by ProbablyIdiot** [comment]
@riverar Sorry, I forgot this wasn't default behaviour.
